appnotes.js

```diff
@@ -0,0 +1,151 @@
+// 'use strict';
+// TOD0: I want prompt the user to guess the number 12 // and get four chances
+
+/* var correctNumber = 12;
```

Comment:
this is an awful lot of commented-out code, you should removed the code and only have comments. it should definitely not be this much.